### PR TITLE
Adds Vespa

### PR DESCRIPTION
**Pull request by AmayaWah**
A ship meant for use within events.
This is a vessel for exploration purposes.
It is meant to be an escape for event-goers. This is a place where events may happen at any time, without warning. People that step foot on this ship must recognise that their RP/ERP will be interrupted by any number of things. Of course, during quiet times, they can do w/e.

This PR adds the Vespa, a prototype courier designed, maintained and owned by Haphaestus Industries.
This vessel comes with a state of the art cloaking device, allowing nimble yet undetectable movement through deep space.
The Vespa is also entirely self-sufficient for a long period of time, allowing expeditions into deep space to last much longer than a normal exploration vessel would permit. This makes it an obvious choice for the exploration crew, but getting to the ship is another thing entirely...

### CODE REVIEW

File: maps/tether/submaps/om_ships/Vespa.dm [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+	base_turf = /turf/simulated/floor/tiled/techfloor
+	landmark_tag = "omship_spawn_aroboat"
+	docking_controller = "aroship_boatbay"
+	shuttle_type = /datum/shuttle/autodock/overmap/HPV
+
+// The 'shuttle'
+/datum/shuttle/autodock/overmap/HPV
+	name = "Vespa's Ship's Boat"
+	current_location = "omship_spawn_aroboat"
+	docking_controller_tag = "aroboat_docker"
```
Comment (Novacat) [inline maps/tether/submaps/om_ships/Vespa.dm]: Here too.

Comment (AmayaWah) [reply maps/tether/submaps/om_ships/Vespa.dm]: Commented out.

File: maps/tether/submaps/om_ships/Vespa.dm [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+	shuttle_tag = "Vespa's Ship's Boat"
+
+
+// A shuttle lateloader landmark
+/obj/effect/shuttle_landmark/shuttle_initializer/HPV
+	name = "Vespa's Boat Bay"
+	base_area = /area/ship/expe/hangar
+	base_turf = /turf/simulated/floor/tiled/techfloor
+	landmark_tag = "omship_spawn_aroboat"
+	docking_controller = "aroship_boatbay"
```
[the remark this answers is not in the document]
Comment (AmayaWah) [reply maps/tether/submaps/om_ships/Vespa.dm]: Commented out.

**Comment by AmayaWah** [comment]
> Since mapmerge is not working, I decided to load this locally to give it a quick look over. Some things I noticed.
> 
> * Way too many engines.
> * Medical, Security, etc. are weirdly stocked. I highly recommend deciding what level/kind of equipment you want for your team, and looking at similarly equipped areas to get an idea of what to get. If you want them to have roughly station level equipment, then I recommend looking at Station medbay/security/engineering/exploration etc. If you want them to have ERT level equipment (which is notably a tier higher than station equipment), then I recommend looking at the ERT Ship and see what that is equipped with.
> * You do not want to give military surfluid shelter capsules to crew. Those give out exceptionally strong gear, equipment, and machines and are generally only supposed to be used by event characters.
> * No cataloguers. Explorers should get these from their locker, and having both on-station explorers and off-station people using them will cause problems.
> * If you are using ballistics, you might want to give the crew access to an unlocked autolathe with plentiful resources to manufacture bullets.
> * You will probably want voidsuits and suit cyclers somewhere.
> * Emergency internals closets and firefighting closets too.
> * Theres a few places where things get tight and will be difficult to navigate. Basically, you want to avoid 1-wide hallways as much as possible or if you have to, keep them as short as possible. Long, 1-wide hallways are a nightmare to navigate.
> * Engineers will not have a whole lot to do, considering that the ship pulls magic power, there is nothing for them to do or setup.
> * Getting to and from your ship will be rather difficult (outside of using portals), since it has no functioning shuttle of its own nor does it have any docking ports for the Tether's expedition shuttle to dock to. Also, because of its sheer size it will not be able to visit anywhere on its own.

changed some things, pls check and review.

**Comment by AmayaWah** [comment]
Hello please review.

**Comment by Novacat** [comment]
There is something really wrong with the diffs for some reason...

**Pull request closed without merging**